package.json

```
@@ -78,7 +78,7 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.0.0",
-    "react": ">=0.14",
+    "react": ">=16.3",
```

this should really use `^`, since it's a major change anyways.
```suggestion
    "react": "^16.3 || ^17",
```